frontend/src/System/Status/About/About.js

```diff
@@ -30,54 +31,54 @@ class About extends Component {
     } = this.props;
 
     return (
-      <FieldSet legend="About">
+      <FieldSet legend={translate('About')}>
         <DescriptionList className={styles.descriptionList}>
           <DescriptionListItem
-            title="Version"
+            title={translate('Version')}
             data={version}
           />
 
           {
             packageVersion &&
               <DescriptionListItem
-                title="Package Version"
-                data={(packageAuthor ? <span> {packageVersion} {' by '} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
+                title={translate('PackageVersion')}
+                data={(packageAuthor ? <span> {packageVersion} {translate('by')} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
               />
           }
 
           {
             isNetCore &&
               <DescriptionListItem
-                title=".Net Version"
+                title={translate('NetCore')}
```

Comment:
Core isn't part of the official name any more.

Reply:
i used NetCore to be in line with radarr, but adjusted to DotNetVersion instead.